.github/dependabot.yml

```diff
@@ -6,7 +6,7 @@ updates:
       interval: "weekly"
       day: "monday"
       time: "06:00"
-      timezone: "UTC"
+      timezone: "Etc/UTC"
```

Comment:
unrelated

Reply:
Reverted in https://github.com/cryptomator/cryptofs/pull/192/commits/60b98e43a14a8b3d7e2bb835307cbef05c4ecf4b